### PR TITLE
Create a new doc page for Main UI‘s special features

### PR DESCRIPTION
**Pull request by florian-h05**
This new page aims at documenting „special“ features of Main UI that really need documentation, like the developer sidebar search (content moved from the tips & tricks section), web audio sink support or the UI Command Item (https://github.com/openhab/openhab-webui/pull/2055).

### CODE REVIEW

**Comment by florian-h05** [comment]
Preview: https://deploy-preview-2188--openhab-docs-preview.netlify.app/docs/ui/mainui.html

@digitaldan FYI this adds the docs for the UI command Item.

File: ui/mainui.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  - script MIME types (requires exact match)
+  - Blockly scripts (using `block`, `blockly` or `blocksource` as search string, where case is ignored)
+  - tags (requires exact match)
+- Persistence Configurations:
+  - label & service id of persistence service
+  - Items persisted by persistence service
+
+## Web Audio Sink
+
+Main UI implements a web audio sink, which means the openHAB server can play audio through on a client device through the browser.
```
Comment (digitaldan) [inline ui/mainui.md:65]:
```suggestion
Main UI implements a web audio sink which allows the openHAB server to play audio through the web browser.
```

**Review by digitaldan — APPROVED** [review]
Thanks @florian-h05 !!!!!

File: ui/mainui.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  Display a notification inside the UI:
+  The `$text` parameter is mandatory, all other parameters are optional.
+  `$closeTimeout` is in milliseconds and defaults 5000, which means by default a notification will be closed after 5 seconds.
+  
+  For example, send `notification:This is the text.:This is the title.:This is the subtitle.:This is the right title.` to the UI command Item to display the following notification:
+  <p align="center"><img style="max-width: 40%;" src="images/notification.png"/></p>
+- `close`:
+  Close all popups, popovers and sheets.
+- `back`:
+  Naviate back.
```
Comment (stefan-hoehn) [inline ui/mainui.md:109]: Navigate

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]

File: settings/aboutpage.md [hunk 3]
```diff
@@ -75,60 +75,8 @@ The first image depicts the normal white setting (off), the second image shows i
 
 - **Disable card expansion animations**: Disables the animations when expanding a widget card.
 - **Disable page transition animations**: Disables the transitions when switching from one page to the other.
-
-### Enable Web Audio sink support
-
-- This setting is also part of the miscellaneous settings.
```
Comment (stefan-hoehn) [inline settings/aboutpage.md:81]: From here on:

I have the feeling you deleted most of what I have written here. Please don't as I have put quite an amount of work by providing context, background and examples into it.

You move around redundant information but don't drop it, thanks.

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]

File: ui/building-pages.md [hunk 4]
```diff
@@ -206,8 +206,7 @@ Substracts one week from the state of `DateTime` and return a relative time repr
 
 ### Debugging Expressions
 
-Expressions can be tested in the Widgets Expression Tester found in the Developer Sidebar
-(<kbd>Shift+Alt+D</kbd>).
+Expressions can be tested in the Widgets Expression Tester found in the [Developer Sidebar]({{base}}/docs/ui/mainui.html#developer-sidebar) (<kbd>Alt</kbd><kbd>Shift</kbd><kbd>D</kbd>).
```
[the remark this answers is not in the document]
Comment (florian-h05) [reply ui/building-pages.md:209]: Fixed (commit not yet pushed).
However this has to be changed anyway when my other PR is merged, we will run into merge conflicts I will have to resolve then.

File: ui/mainui.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+Currently, the supported commands are:
+
+- `navigate:$path`:
+  Navigate to a given `$path`, e.g. `navigate:/locations` to navigate to the "Locations" tab of the home page or `navigate:/page/my_custom_page` to navigate to a page with the ID `my_custom_page`.
+- `popup:$target`, `popover:$target` and `sheet:$target`:
+  Open a `$target`, which can be a page (`$target` is `page:` + a page ID), a custom widget (`$target` is `widget:` + widget ID) or any [`oh-` component]({{base}}/ui/components/) (`$target` is the component name), in a popup, popover or sheet.
+  
+  For example, to open `oh-clock-card` inside a popup, send `popup:oh-clock-card` to the UI command Item.
+- `notification:$text:$title:$subtitle:$titleRight:$closeTimeout`:
```
[the remark this answers is not in the document]
Comment (stefan-hoehn) [reply ui/mainui.md:100]: Then would you mind adding this as an example or mention this?

[the remark this answers is not in the document]
Comment (florian-h05) [reply ui/mainui.md:100]: I’ve added this sentence here:

> To leave out a parameter, leave a colon space empty, e.g. `notification:$text:$title:::$closeTimeout`.

Should I still add an example?

[the remark this answers is not in the document]
Comment (stefan-hoehn) [reply ui/mainui.md:100]: No, that's fine.

File: ui/building-pages.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (florian-h05) [reply ui/building-pages.md]: I have just rebased the PR, this comment is resolved since the section doesn't exist anymore on that page.

**Comment by florian-h05** [comment]
Superseded by #2198.

**Pull request closed without merging**